CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.3.0 (09/15/2016)
```

Comment:
I would be incredibly surprised if anyone ran into it, which is why I didn't put it. more likely to confuse people since it's a bit complicated. I could add it though.